ruby/ql/src/queries/security/cwe-116/examples/BadTagFilter.rb

```diff
@@ -0,0 +1,8 @@
+def filterScripTags(html)
+  oldHtml = "";
+  while (html != oldHtml)
+    oldHtml = html;
+    html = html.gsub(/<script[^>]*>.*<\/script>/m, "");
+  end
+  return html;
```

Comment:
Just some suggestions to make the Ruby code a little more idiomatic.

```suggestion
def filter_script_tags(html)
  old_html = ""
  while (html != old_html)
    old_html = html
    html = html.gsub(/<script[^>]*>.*<\/script>/m, "")
  end
  html
```